### PR TITLE
Upadte/domain transfer faqs

### PR DESCRIPTION
**Pull request by davemart-in**
## Description

Made a handful of follow up updates based on feedback. 

## Changes

- "It looks like I'm being charged, I thought the first year was free?" question
- Make name server plural & add clarity as outlined by Wendy
- [Font size tweak](https://github.com/Automattic/wp-calypso/pull/79503#issuecomment-1640406453) from Nuria

## Preview

CleanShot 2023-07-18 at 11 39 11@2x

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/faqs.tsx [hunk 1]
```diff
@@ -138,6 +140,32 @@ const DomainTransferFAQ: FC = () => {
...
+						<FoldableFAQ
+							id="privacy-protection"
+							question={ translate(
+								'It looks like I am being charged, I thought the first year was free?'
+							) }
+							onToggle={ onFaqToggle }
+							className="domain-transfer-faq__section"
+						>
+							{ translate(
+								'We offer a free first year for all domains being transferred in, but some domain extensions may not ' +
```
Comment (ebinnion) [inline client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/faqs.tsx]: This offer only applies to domains that are coming from Google Domains.

Comment (davemart-in) [reply client/landing/stepper/declarative-flow/internals/steps-repository/domain-transfer-domains/faqs.tsx]: 👍

**Review by ebinnion — APPROVED** [review]

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/8598275

Thank you @davemart-in for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/8598275).

**Pull request merged**